modules/cyclone-dx/src/test/resources/logback.xml

```diff
@@ -0,0 +1,24 @@
+<!--
```

Comment:
We have configured logging lately, is this additional configuration really necessary?

Reply:
Not any longer. Is normally to avoid ugly warnings from slf4j when not configured. Will remove it.

Reply:
Again, did this come back? From your answer I expected, that this is not needed anymore?